storybookjs/storybook
# #2081 #2117 fix prop type in react-native

Issue:

#2081 #2117
There is an optional prop was listed as required

## What I did

Changed it to optional and added a defaultProp

## How to test

Is this testable with jest or storyshots?
yes, passes the test with no changes
Does this need a new example in the kitchen sink apps?
no
Does this need an update to the documentation?
no
If your answer is yes to any of these, please make sure to include it in your PR.
